.github/workflows/ci.yaml

```diff
@@ -15,8 +15,6 @@ jobs:
       matrix:
         os:
           - ubuntu-20.04
-          - ubuntu-18.04
```

Comment:
Can you explain this one? Are Ubuntu 18 and 16 considered too old to support you think?

Reply:
well, I just don't see the point since we are using the same toolchain for both. Do we ever see one fail and not the other for any meaningful reason? or are we just warming the earth with this one.

Reply:
It might be a libncurses version issue.  I guess we can fix it with package dependencies, but it some versions of ubuntu have libncurses6 and some have libncurses5 and when we link against one or the other and it's missing, that's when we get the libtinfo errors